etc/testing/port-forward.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
The web server tests don't need to run in k8s, but now they rely on postgres which is deployed on k8s. This script port-forwards postgres from k8s so the tests can use it when they're running outside, which makes iteration must faster.